requirements.txt

```diff
@@ -3,3 +3,5 @@
 # SPDX-License-Identifier: Unlicense
 
 Adafruit-Blinka
+board
+serial
```

Comment:
I think `board` and `serial` don't need to be added to requirements file here.

Instead of `board.UART` for the UART type we want to use `busio.UART` so we won't need to import board any more once that is changed.

For both `busio` and `board` (if it were used) they would come from `Adafruit-Blinka` which is already listed in here (albeit under a different name.)

`serial` I thought was a built-in module, not something installed from pip, maybe I've got it confused with something though. If that does need to come from pip we might use it in optional requirements instead of regular, or perhaps consider making a protocol definition for it in circuitpython_typing library to avoid adding an external dependency for the typing only.